package.json

```diff
@@ -10,12 +10,20 @@
     "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
   },
   "devDependencies": {
-    "axios": "^0.16.2",
+    "axios": "^0.18",
     "bootstrap-sass": "^3.3.7",
-    "cross-env": "^5.0.1",
-    "jquery": "^3.1.1",
-    "laravel-mix": "^1.0",
-    "lodash": "^4.17.4",
-    "vue": "^2.1.10"
+    "cross-env": "^5.1",
+    "jquery": "^3.2",
+    "laravel-mix": "^4.0.15",
+    "lodash": "^4.17.5",
+    "resolve-url-loader": "^2.3.1",
+    "sass": "^1.20.1",
+    "sass-loader": "^7.1.0",
+    "tailwindcss": "^1.0.1",
+    "vue": "^2.6.7",
+    "vue-template-compiler": "^2.6.10"
+  },
+  "dependencies": {
+    "vform": "^1.0.0"
```

Comment:
vform is used by the Tailwind-Vue-Components I used, so I decided to stick with theirs rather than fighting it and re-writing it on my own